docs/dev/table/sql/queries.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Queries"
+title: "SELECT queries"
```

Comment:
In this section, Do we need change this? all queries should be SELECT queries from my understanding. Others should belong to other kind of SQL, "query" should not equals "SQL"

Reply:
change to `SELECT statements` which style is the same as others